app/controllers/cars_controller.rb

```diff
@@ -0,0 +1,62 @@
+class CarsController < ApplicationController
+  # GET /cars
+  def index
+    @cars = Car.all
+    render json: @cars
+  end
+
+  # GET /cars/1.json
+  def show
+    @car = Car.find(params[:id])
+    render json: @car
+  end
+
+  # GET /cars/new
+  def new; end
+
+  # GET /cars/1/edit
+  def edit; end
```

Comment:
- [ ] I don't think you need to declare empty methods if you're not using them, so I suggest you remove them.